To: vim_dev@googlegroups.com
Subject: Patch 7.3.153
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.153 (after 7.3.152)
Problem:    Compiler warning for ambiguous else, missing prototype.
Solution:   Add braces. (Dominique Pelle)  Add prototype for die().
Files:	    src/xxd/xxd.c


*** ../vim-7.3.152/src/xxd/xxd.c	2011-04-01 19:14:35.000000000 +0200
--- src/xxd/xxd.c	2011-04-02 14:42:54.000000000 +0200
***************
*** 210,215 ****
--- 210,216 ----
  /* Let's collect some prototypes */
  /* CodeWarrior is really picky about missing prototypes */
  static void exit_with_usage __P((void));
+ static void die __P((int));
  static int huntype __P((FILE *, FILE *, FILE *, int, int, long));
  static void xxdline __P((FILE *, char *, int));
  
***************
*** 689,694 ****
--- 690,696 ----
  
  	  while (s--)
  	    if (getc(fp) == EOF)
+ 	    {
  	      if (ferror(fp))
  		{
  		  die(2);
***************
*** 698,703 ****
--- 700,706 ----
  		  fprintf(stderr, "%s: sorry cannot seek.\n", pname);
  		  return 4;
  		}
+ 	    }
  	}
      }
  
*** ../vim-7.3.152/src/version.c	2011-04-01 19:14:35.000000000 +0200
--- src/version.c	2011-04-02 14:41:12.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     153,
  /**/

-- 
            |

Ceci n'est pas une pipe.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
